envs/arrow-env.yml

```diff
@@ -0,0 +1,38 @@
+packages:
+{% if s390x %}
+  - feedstock : https://github.com/AnacondaRecipes/rapidjson-feedstock
+    git_tag: 3ccf90746229705d0802944babd62fe8fadb7d19
+  - feedstock : https://github.com/AnacondaRecipes/orc-feedstock
+    git_tag: 47b595ae87b7bd10b16e08aef41e1b9e9d257848
+  - feedstock : https://github.com/conda-forge/libutf8proc-feedstock
+    git_tag: d2a2366dc3c02405b1b54b991758b5bb57b75e6d
+  - feedstock : https://github.com/AnacondaRecipes/re2-feedstock
+    git_tag: 428e7197734e4b3ca835474e31ada8c8c129adee
+  - feedstock : https://github.com/AnacondaRecipes/abseil-cpp-feedstock
+    git_tag: 54375be823fd7e84d8a4bfc8cc1f2718624d0d6e
+  - feedstock : https://github.com/AnacondaRecipes/grpc-cpp-feedstock
+    git_tag: e0e787b0ffbddd7e1faeba6157a8d90f2499d567
+    patches:
+      - ../feedstock-patches/grpc_cpp/s390x-fix1.patch
+  - feedstock : https://github.com/AnacondaRecipes/gflags-feedstock
+    git_tag: 8bfee448e77492920c23780385dddc5982947e58
+  - feedstock : https://github.com/AnacondaRecipes/glog-feedstock
+    git_tag: 4169d3a5f6238537cfcd96fb7b278e1f322fbae7
+    patches:
+      - ../feedstock-patches/glog/s390x-fix1.patch
+  - feedstock : https://github.com/conda-forge/double-conversion-feedstock
+    git_tag: 4b8aebe28cdb4d7215bccbf4c0e76f21fc0123f0
+  - feedstock : https://github.com/AnacondaRecipes/boost-cpp-feedstock
+    git_tag: 95c6649b96a11fb439d04fe36cc645720d156afb
+    patches:
+      - ../feedstock-patches/boost_cpp/build-only-required.patch
+  - feedstock : https://github.com/AnacondaRecipes/libevent-feedstock
```

Comment:
Is there a reason this is being pulled from AnacondaRecipes instead of the Open-CE feedstock?

Reply:
open-ce is having a slightly newer version of libevent than ancondaRecipes and the opence based libevent feedstock's build fails on s390x saying 1 out 299 tests failed.
libevent 2.1.10 in s390x Vs 2.1.11 in opence.
Ah! Tests are disabled in AnacodnaRecipes with a comment "Seems to hang on Mac builds. So have commented it for now."
Let me give a try of tests on s390x with 2.1.10 once

Reply:
With tests enabled in AnacondaRecipes (libevent version 2.1.10), conda build seems to be hanging. This behavior is similar to what is mentioned in the comment about osx side.
For s390x, if we start using the newer version (2.1.11)'s feedstock from opence, we have to fix the test failure (1 out of 299 tests failed on s390x) OR have to disable tests for s390x.